Source/Controllers/MainViewControllers/SPDatabaseDocument.m

```diff
@@ -5572,7 +5572,9 @@ - (void)_copyDatabaseWithDetails:(NSDictionary *)databaseDetails
 		[self endTask];
 
 		if (!success) {
-			[NSAlert createWarningAlertWithTitle:NSLocalizedString(@"Unable to copy database", @"unable to copy database message") message:[NSString stringWithFormat:NSLocalizedString(@"An error occured while trying to copy the database '%@' to '%@'.", @"unable to copy database message informative message"), [databaseDetails[SPNewDatabaseDetails] databaseName], newDatabaseName] callback:nil];
+			dispatch_sync(dispatch_get_main_queue(), ^{
```

Comment:
Could use `SPMainQSync` from `SPFunctions.m`?

Reply:
Yeah I'm not a big fan of these... they could be pain in the *** when converting to swift. 🙈